core/lib/eth_client/src/clients/http/signing.rs

```diff
@@ -217,6 +217,8 @@ impl<S: EthereumSigner> BoundEthInterface for SigningClient<S> {
         data: Vec<u8>,
         contract_addr: H160,
         options: Options,
+        max_fee_per_blob_gas: Option<U256>,
```

Comment:
shouldn't we check that if blob_versioned_hashes is None, than this is also None?